packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ProxyEOA.sol

```diff
@@ -22,48 +20,34 @@ contract OVM_ProxyEOA {
      * Constants *
      *************/
 
+    address constant DEFAULT_IMPLEMENTATION = 0x4200000000000000000000000000000000000003;
     bytes32 constant IMPLEMENTATION_KEY = 0xdeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddead;
 
 
-    /***************
-     * Constructor *
-     ***************/
-
-    /**
-     * @param _implementation Address of the initial implementation contract.
-     */
-    constructor(
-        address _implementation
-    )
-    {
-        _setImplementation(_implementation);
-    }
-
-
     /*********************
      * Fallback Function *
      *********************/
 
     fallback()
         external
     {
-        (bool success, bytes memory returndata) = Lib_SafeExecutionManagerWrapper.safeDELEGATECALL(
-            gasleft(),
-            getImplementation(),
-            msg.data
-        );
+        (bool success, bytes memory returndata) = getImplementation().delegatecall(msg.data);
 
         if (success) {
             assembly {
                 return(add(returndata, 0x20), mload(returndata))
             }
         } else {
```

Comment:
This success case doesn't seem to match the one in `_safeExecutionManagerInteraction`:
https://github.com/ethereum-optimism/optimism/blob/17654d06a034b655243d63a27ecbf21da290a255/packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_SafeExecutionManagerWrapper.sol#L350-L360

Reply:
Ah this contract is compiled with optimistic-solc now so it doesn't need to do the manual safety checks that we did in `Lib_SafeExecutionManagerWrapper`.